cmake/projects/OpenCV/hunter.cmake

```diff
@@ -12,6 +12,21 @@ include(hunter_download)
 include(hunter_pick_scheme)
 
 # List of versions here...
+
+hunter_add_version(
+    PACKAGE_NAME
+    OpenCV
+    VERSION
+    "4.1.1"
+    URL
+	"https://github.com/kumaakh/hunter-opencv/archive/hunter-4.1.1.zip"
```

Comment:
Let’s get the PR against the cpp-pm fork merged first, and then we can release that and update this link to be the cpp-pm release